huaweicloud/resource_huaweicloud_bcs_instance_test.go

```diff
@@ -0,0 +1,310 @@
+package huaweicloud
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/terraform"
+
+	"github.com/huaweicloud/golangsdk"
+	"github.com/huaweicloud/golangsdk/openstack/bcs/v2/blockchains"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/config"
+)
+
+//Huawei@123
+const salt_password string = "JDYkcGxkUm9uUE55UTEweGRCUyRxbWFDRFluTFhPVjNCakVraGc5YUdhUTBxeWFocmNDbzZ4YS9rVWkyYVB2S" +
```

Comment:
hard coding is not recommended, can we use key_pair?

Reply:
It's updated, can we support random provider in acc test?